spec/lib/govuk_publishing_components/components/component_wrapper_helper_spec.rb

```diff
@@ -118,16 +118,16 @@
 
     describe "setting an id" do
       it "does not accept invalid ids" do
-        ["1dstartingwithnumber", "id with spaces", "idwith.dot", "id\nwithnewline"].each do |id|
+        ["id with spaces", "idwith.dot", "id\nwithnewline"].each do |id|
           expect {
             GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(id:)
           }.to raise_error(ArgumentError, / contain/)
```

Comment:
Just curious, is there a reason this doesn't match the exact error string whereas other tests do?

Reply:
I think because the error message changes each time, as it outputs the id in the error message.

Reply:
Ah OK - though it seems the other tests account for that, e.g. the other test is`"Classes (#{classes}) must be prefixed with `js-`"` - but this a non blocking nit pick thing anyway so feel free to ignore it.

Reply:
Oddly I don't recognise this code and it doesn't appear elsewhere in the tests, but always interesting to see different ways of doing the same thing. Thanks.